NIPTool/API/external/api/api_v1/endpoints/sample.py

```diff
@@ -30,7 +37,12 @@ def sample(request: Request, sample_id: str, adapter: NiptAdapter = Depends(get_
 
 
 @router.post("/samples/{sample_id}/")
-def sample(request: Request, sample_id: str, adapter: NiptAdapter = Depends(get_nipt_adapter)):
+def sample(
+    request: Request,
+    sample_id: str,
+    adapter: NiptAdapter = Depends(get_nipt_adapter),
+    user=Depends(get_current_user),
+):
     """Post sample with id"""
 
     sample: dict = find.sample(sample_id=sample_id, adapter=adapter).dict()
```

Comment:
```suggestion
    sample: Sample = find.sample(sample_id=sample_id, adapter=adapter).dict()
```

Reply:
This needs to be a dict in this branch. IN my models pr the utils functions take Sample object and not dict